### ISSUE TITLE
status and StatusEnumeration don't match in netex_version_support-v1.1.xsd

### ISSUE THREAD

**Issue by ue71603** [NeTEx-CEN/NeTEx#169]
Defaultstatus should be "released", which does not exist:
		<xsd:attribute name="status" type="StatusEnumeration" use="optional" default="active">
			<xsd:annotation>
				<xsd:documentation>Whether ENTITY is currently in use. Default is "released"</xsd:documentation>
			</xsd:annotation>
		</xsd:attribute>

	<xsd:simpleType name="StatusEnumeration">
		<xsd:annotation>
			<xsd:documentation>Indicates whether the ENTITY this annotates is available for use. Use of this attribute allows entities to be retired without deleting the details from the dataset.</xsd:documentation>
		</xsd:annotation>
		<xsd:restriction base="xsd:NMTOKEN">
			<xsd:enumeration value="active">
				<xsd:annotation>
					<xsd:documentation>Entity is active.</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="inactive">
				<xsd:annotation>
					<xsd:documentation>Entity is inactive.</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="other">
				<xsd:annotation>
					<xsd:documentation>Entity is still active but is in the process of being made inactive.</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
		</xsd:restriction>
	</xsd:simpleType>

![image](https://user-images.githubusercontent.com/24227470/116539853-ae4c1480-a8e9-11eb-8a98-5896f90f0850.png)

In the specification it could not be linked up. There a different element and Enum is used (EntityInVersion):
![image](https://user-images.githubusercontent.com/24227470/116540136-084cda00-a8ea-11eb-8f32-53d95cc82e6f.png)

The current master link is here: C:\Users\ue71603\MG_Daten\netex_sncf_2018-12\NeTEx-V01.09\xsd\netex_framework\netex_responsibility\netex_version_support-v1.1.xsd

**Comment by skinkie** [comment]
@ue71603 as per document, please add a screenshot from the documentation as well.

**Comment by Aurige** [comment]
Isn't **_released_** something different ? it sounds more like a publishing status to me, the **_StatusEnumeration_** enumeration is about the real physical statu,  which can be different from the publishing one.
That's true that we don't have publication status in NeTEx... maybe this has to be linked to this discussion: https://3.basecamp.com/3256016/buckets/2570434/messages/3703354430#__recording_3721468173  ?

**Comment by Aurige** [comment]
Following this discussion I'm afraid we now have a small inconstent comment in the XSD now in _netex_passengerInformationEquipment_version.xsd_
```
		<xsd:attribute name="status" type="StatusEnumeration" use="optional" default="active">
			<xsd:annotation>
				<xsd:documentation>Whether ENTITY is currently in use. Default is "**released**"</xsd:documentation>
			</xsd:annotation>
		</xsd:attribute>
```
But **released** is not in the enum

**Comment by ue71603** [comment]
fixed in #583
